packages/docusaurus/src/client/exports/BrowserOnly.tsx

```diff
@@ -15,11 +16,13 @@ function BrowserOnly({
   children?: () => JSX.Element;
   fallback?: JSX.Element;
 }): JSX.Element | null {
+  const {isClient} = useDocusaurusContext();
+
   if (!ExecutionEnvironment.canUseDOM || children == null) {
```

Comment:
I'd rather use 

```
  if (!isClient || !ExecutionEnvironment.canUseDOM || children == null)
```

We need to have the exact same markup on server and on first client render, so this is not good enough, as it does not include the potential fallback (not always null): `isClient ? <>{children()}</> : null;`

Reply:
Ah, yes, you are right, just fixed that.